src/i18n/pl_PL.ts

```diff
@@ -0,0 +1,1050 @@
+            special:
+                'Przy posiadaniu powyżej 24 budynków straży pożarnej koszt budowy wzrasta według wzoru:<code>100.000+200.000*LOG<sub>2</sub>(Liczba istniejących straży pożarnej â’ 22)</code>, max 1 milion kredytów. Cena za Monety pozostaje bez zmian!',
+            startPersonnel: 10,
+            startVehicles: ['Ciężki samochód gaśniczy', 'Średni samochód gaśniczy', 'GBARt', 'GLBA'],
+            maxBuildingsFunction: (): number => 4_000,
+        },
+        19: {
+            caption: 'Posterunek Policji',
+            color: '#93B7FF',
+            coins: 25,
+            credits: 50_000,
+            extensions: [
+                {
+                    caption: 'Cela więzienna',
+                    credits: 25_000,
+                    coins: 5,
+                    duration: '7 Dni',
+                },
+                ...new Array(1).fill({
+                    caption: 'Dodatkowa cela',
+                    credits: 25_000,
+                    coins: 5,
+                    duration: '7 Dni',
+                }),
+            ],
+            levelcost: [
+                '1. 10.000',
+                '2. 50.000',
+                '3.-4. 100.000',
+                'Cena przebudowy na Komendę Policji wynosi różnicę w cenach budynków ',
+            ],
+            maxBuildings: '1.500 włącznie z Komendami Policji',
+            maxLevel: 4,
+            special:
+                'Przy posiadaniu powyżej 24 budynków policji koszt budowy wzrasta według wzoru:<code>100.000+200.000*LOG<sub>2</sub>(Liczba istniejących budynków policji â’ 22)</code>. Cena za Monety pozostaje bez zmian!',
+            startPersonnel: 2,
+            startVehicles: ['Radiowóz OPI'],
+            maxBuildingsFunction: (): number => 1_500,
+        },
+        20: {
+            caption: 'Podstacja Pogotowia Ratunkowego',
+            color: '#FFFFCC',
+            coins: 25,
+            credits: 100_000,
+            extensions: [],
+            levelcost: [
+                '1. 10.000',
+                '2. 50.000',
+                '3.-5. 100.000',
+                'Cena przebudowy na Stację Pogotowania Ratunkowego wynosi różnicę w cenach budynków ',
+            ],
+            maxBuildings: 'Bez limitu',
+            maxLevel: 5,
+            special: '',
+            startPersonnel: 3,
+            startVehicles: ['Ambulans P'],
+        },
+    },
+    buildingCategories: {
+        'Straż pożarna': {
+            buildings: [0, 1, 18],
+            color: '#990000',
+        },
+        'Pogotowie ratunkowe': {
+            buildings: [2, 3, 5, 20],
+            color: '#FFFFCC',
+        },
+        'Policja': {
+            buildings: [6, 8, 13, 19],
+            color: '#93B7FF',
+        },
+        'WOPR': {
+            buildings: [3, 15],
+            color: '#F9D74A',
+        },
+        'Inne': {
+            buildings: [7, 4, 14],
+            color: '#02a18c',
+        },
+    },
+    vehicleCategories: {

```

Comment:
Please translate the category names and vehicle groups. Did you already checked these vehicle ID's?